modules/environment-roles/templates/shared_terraform_policy_2.json.tpl

```diff
@@ -82,7 +82,14 @@
         "arn:aws:iam::${account_id}:role/TDRLogDataAssumeRole${environment}",
         "arn:aws:iam::${account_id}:role/TDRLogDataCrossAccountRoleMgmt",
         "arn:aws:iam::${account_id}:policy/TDRApiUpdatePolicy",
-        "arn:aws:iam::${account_id}:role/TDRApiUpdateRole"
+        "arn:aws:iam::${account_id}:role/TDRApiUpdateRole",
+        "arn:aws:iam::${account_id}:role/FileFormatEcsTaskRole${environment}",
```

Comment:
New roles are missing the "TDR" prefix. Maybe this should be added for consistency.

Reply:
I've changed these, they match the names in the modules repo now.